collectors/python.d.plugin/hpssa/hpssa.chart.py

```diff
@@ -234,6 +236,15 @@ def parse_unassigned_physical_drives(self, adapter):
                 self.rewind()
                 break
 
+    def parse_hba_physical_drives(self, adapter):
+        for line in self:
+            match = drive_regex.match(line)
+            if match:
+                self.parse_physical_drive(adapter, match.group('fqn'))
+            else:
+                self.rewind()
+                break
+
```

Comment:
i see this function equal to `parse_unassigned_physical_drives`

https://github.com/martinpal/netdata/blob/cc34080dcf10fcc0d74ea4cd81e53193646f383d/collectors/python.d.plugin/hpssa/hpssa.chart.py#L230-L246

How about to call it `parse_physical_drives` and use for both `line == 'Unassigned'` and `line == 'HBA Drives'`?

Reply:
That would be better indeed. Updated. That makes my minor contribution even smaller - a one-line change.